test/parallel/test-repl-history-navigation.js

```diff
@@ -54,6 +54,7 @@ const LEFT = { name: 'left' };
 const RIGHT = { name: 'right' };
 const DELETE = { name: 'delete' };
 const BACKSPACE = { name: 'backspace' };
+const TABULATION = { name: 'tab' };
```

Comment:
Why was this test added, it doesn't seem relevant to the change

Reply:
By reading the source code, this section is to add test case for the modified code